Wikipedia/Code/NotificationsCenterDetailViewController.swift

```diff
@@ -31,6 +31,18 @@ final class NotificationsCenterDetailViewController: ViewController {
         detailView.tableView.dataSource = self
         detailView.tableView.delegate = self
     }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupNavigationBar()
+    }
+    
+    private func setupNavigationBar() {
+        
+        let detailTitle = WMFLocalizedString("notifications-center-detail-title", value: "Notification Detail", comment: "Title of notification detail view, displayed after tapping a notification in Notifications Center.")
```

Comment:
Just wanted to note, the long press previous view menu item title on pushed views changes to Notification Detail, but the back button title itself remains Back. Not suggesting any change, as this is an improvement from existing functionality which just displays "Back" as both the long press option and the back button title.
![detail](https://github.com/wikimedia/wikipedia-ios/assets/5652327/c2368cbc-279a-46f2-a1b8-a3fedc8d28c0)